string hostname
string ipaddress

time window_start
time window_stop

int16 cpus_available

uint16 num_samples

float32[] cpu_load_mean
float32[] cpu_load_std
float32[] cpu_load_max

# mem_avialable
float64 phymem_used_mean
float64 phymem_used_std
float64 phymem_used_max
float64 phymem_avail_mean
float64 phymem_avail_std
float64 phymem_avail_max
